RELEASE_NOTES.md

```diff
@@ -267,6 +267,16 @@ Please read more in the docs:
 
 You can check more about the implementation in the [\#10700](https://github.com/decidim/decidim/pull/10700) pull request.
 
+### 3.11 Patch robots.txt
+
+In order to append default Decidim robots.txt to the application's robots.txt, we have added a new task that helps you to do that.
```

Comment:
```suggestion
In order to improve the fight against spam attacks in Decidim applications, we have added a new task that helps you replace yours. Take into account that this will override your robots.txt, so if you have done any change you need to make a backup before running this task.
```